hm_pyhelper/hardware_definitions.py

```diff
@@ -138,9 +138,9 @@ def is_raspberry_pi():
         'CELLULAR': False
         },
 
-    # Nebra Hotspot HAT Rockpi 4
+    # Nebra Hotspot HAT Rockpi 4 Indoor
     'NEBHNT-HHRK4': {
-        'FRIENDLY': 'Nebra Hotspot HAT RockPi 4',
+        'FRIENDLY': 'Nebra RockPi 4 Indoor',
```

Comment:
`RockPi`: We are inconsistent with this spelling in various places. Their website uses ROCK Pi a lot. Is this what we should use everywhere?

Reply:
@marvinmarnold  I'm not fussed either way to be honest. 

I thought it was RockPi just because it's based on Rockchip processor. 

But as you say it says ROCK Pi on their website everywhere so shall we go with that and try to unify it across all areas?

Reply:
Yea, might as well use that and unify. Obv not a huge deal.